helper_apps/MediaFileChunker/blobHandler.js

```diff
@@ -0,0 +1,151 @@
+import fs from 'fs';
+import path from 'path';
+import { BlobServiceClient } from '@azure/storage-blob';
+import { v4 as uuidv4 } from 'uuid';
+import Busboy from 'busboy';
+import { PassThrough } from 'stream';
+import { pipeline as _pipeline } from 'stream';
+import { promisify } from 'util';
+const pipeline = promisify(_pipeline);
+import { join } from 'path';
+
+
+import { publicFolder, port, ipAddress } from "./start.js";
+
+const getBlobClient = () => {
+    const connectionString = process.env.AZURE_STORAGE_CONNECTION_STRING;
+    const containerName = process.env.AZURE_STORAGE_CONTAINER_NAME;
+    if (!connectionString || !containerName) {
+        throw new Error('Missing Azure Storage connection string or container name environment variable');
+    }
+
+    const blobServiceClient = BlobServiceClient.fromConnectionString(process.env.AZURE_STORAGE_CONNECTION_STRING);
```

Comment:
Why process.env here if you have done the assign above?

Reply:
correct, i did so many back and forths basically left unneeded code coming from even my first pass. normally i do a clean up pass this time i forgot